### PR TITLE
Code changes to add various error messages to NW Workbook

### PR DESCRIPTION
**Pull request by hsridharan**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

This PR is for adding validation error messages to the NW Workbook. Please follow the below document to go through the scenarios: https://microsoft-my.sharepoint.com/:w:/r/personal/hsridharan_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8BFF312C-792E-46B7-89DF-16EC33D873CE%7D&file=WorkbookErrors.docx&wdLOR=c65B433D3-EA44-4401-9711-490A55A97664&nav=eyJjIjo5NzAzNjI0MH0&action=default&mobileredirect=true

### CODE REVIEW

**Review by rohPatil — APPROVED** [review]

File: Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook [hunk 1]
```diff
@@ -8,6 +8,143 @@
...
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "conditionalVisibility": {
+        "parameterName": "dummy_neverset",
+        "comparison": "isNotEqualTo"
```
Comment (MikeFrei) [inline Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook:75]: are you missing a "value" element here?

File: Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook [hunk 2]
```diff
@@ -602,237 +751,641 @@
...
+      },
+      "conditionalVisibilities": [
+        {
+          "parameterName": "selectedMenu",
+          "comparison": "isEqualTo",
+          "value": "Availability"
+        },
+        {
+          "parameterName": "dummy_neverset",
+          "comparison": "isNotEqualTo"
```
Comment (MikeFrei) [inline Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook:801]: same comment here, are you missing a "value" property here?

File: Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook (hunk 1 above)
Comment (hsridharan) [reply Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook:75]: No this is a dummy element added for some calculations which will be never set.

File: Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook [hunk 3]
```diff
@@ -602,237 +751,641 @@
...
         "version": "NotebookGroup/1.0",
         "groupType": "editable",
         "items": [
           {
-            "type": 9,
+            "type": 1,
             "content": {
-              "version": "KqlParameterItem/1.0",
-              "parameters": [
+              "json": "Unable to create the required table (GetProcessList) to capture telemetry information. Please try after some time. If the issue persists, please contact support.",
```
Comment (MikeFrei) [inline Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook:815]: wording here seems off to me.  These workbooks themselves are never actually creating any tables right?  It seems like what we really want to say is that the expected table has not been created yet, which could indicate that data has never been uploaded.  Actually, what is the difference between this message and the message below?  In both cases the table does not exist, right?

Comment (MikeFrei) [reply Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook:815]: per discussion, harsha was just following Hana provider convention here.

**Review by bluereign — APPROVED** [review]

**Review by MikeFrei — APPROVED** [review]

**Pull request merged**